jerry-core/api/jerry-snapshot.h

```diff
@@ -51,8 +51,6 @@ typedef enum
   /* 8 bits are reserved for dynamic features */
   JERRY_SNAPSHOT_HAS_REGEX_LITERAL = (1u << 0), /**< byte code has regex literal */
   JERRY_SNAPSHOT_HAS_CLASS_LITERAL = (1u << 1), /**< byte code has class literal */
-  /* 24 bits are reserved for compile time features */
-  JERRY_SNAPSHOT_FOUR_BYTE_CPOINTER = (1u << 8) /**< compressed pointers are four byte long */
```

Comment:
Is it OK to simply drop this enumerator? In other public API headers (e.g., jerryscript-core.h), there are examples to keep deprecated enumerators with original values, with a comment stating that it's "deprecated, an unused placeholder now" (see `jerry_init_flag_t`).

Reply:
Thank you for the clarification. I've modified the patch and added a deprecated marker to this entry.